src/main/java/com/arpnetworking/tsdcore/sinks/PeriodicStatisticsSink.java

```diff
@@ -105,6 +113,24 @@ public Object toLogValue() {
                 .build();
     }
 
+    static Key computeKey(final Key key, final ImmutableMultimap<String, String> mappedDimensions) {
```

Comment:
Maybe make a note about how this "flattens" the mappedDimensions and only keeps the last one?